packages/dd-trace/src/appsec/iast/index.js

```diff
@@ -0,0 +1,66 @@
+const { sendVulnerabilities } = require('./vulnerability-reporter')
+const { enableAllAnalyzers, disableAllAnalyzers } = require('./analyzers')
+const web = require('../../plugins/util/web')
+const { storage } = require('../../../../datadog-core')
+const overheadController = require('./overhead-controller')
+const dc = require('diagnostics_channel')
+const { saveIastContext, getIastContext, cleanIastContext } = require('./iast-context')
+
+const requestStart = dc.channel('dd-trace:incomingHttpRequestStart')
```

Comment:
What event do you mean? 
nodejs built in `http.server.request.start` is not available at least until node 15.

If you are talking about `apm:http:server:request:start`, when it is launched, store and span are not initialized yet.

Reply:
If we order the channels properly the span should be available. Since everything relies on a span being active, it makes sense to always run tracing subscribers first so that it's available for everything else.

Reply:
Ah yes because of lazy loading? There is no guarantee that require calls are synchronous, so a timer wouldn't work.

Actually does it even make sense to load appsec in the first place if `http` is not even used? Maybe it could add its subscribers only when that happens? Or maybe it could listen to tracing events instead of library events, although those don't exist yet.

Reply:
Ok, @simon-id if you are agree, 'm going to change it to **iast** and also to **appsec**
- [ ] Subscribe to `apm:http:server:request:start` in the correct order in appsec and iast

Reply:
Right that's true, for now AppSec only act in tendem with the http module.@uurien Please don't change anything in AppSec for now, as I'm already in the process of changing it in another PR.

Reply:
Ok, changed only for IAST

Reply:
> Actually does it even make sense to load appsec in the first place if http is not even used?
Actually no, but in the near future, we will be able to detect some vulnerabilities out of the request.

I fixed it with the @simon-id proposed `setImmediate()` but I'm not happy with this solution, if some change in initialization changes the order, we would stop working.
But with the original `dd-trace:incomingHttpRequestStart` the solution was consistent.

Reply:
Let's not use timers for this. If there is no way to enforce the order we need just keep the current events and add a TODO.